src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -57,11 +57,7 @@
 import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
 import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
 import org.glassfish.hk2.api.MultiException;
```

Comment:
It seems this code is still dependent on `org.glassfish.hk2.api.MultiException` and `org.glassfish.jersey.internal.util.Base64`. Are you sure that this client still works when glassfish has completely vanished from the classpath? And that the exception handling works well if `glassfish` is replaced with `resteasy`?